fe/fe-core/src/main/java/com/starrocks/catalog/EsTable.java

```diff
@@ -202,6 +203,14 @@ private void validate(Map<String, String> properties) throws DdlException {
         }
         hosts = properties.get(KEY_HOSTS).trim();
         seeds = hosts.split(",");
+        for (int i = 0; i < seeds.length; i++) {
+            seeds[i] = seeds[i].trim();
+        }
+        for (String seed : seeds) {
+            if (!seed.startsWith("http://") && !seed.startsWith("https://")) {
+                throw new DdlException("Host of ES table should start with 'http:// or 'https://'. Current value is " + seed);
```

Comment:
I'm wondering why the prefix of http and https is necessary.
Is there any possibility that someone has already has some property without "http" or https as prefix, but after they upgrade to the new version, the existing ES external table can work as normal.

Reply:
![image](https://github.com/StarRocks/starrocks/assets/1081215/85bf3898-cb32-4862-adff-5fe7a347cf0a)

I've posted that in description. That's to say, if you don't prefix with http:// or https://, port will be wrong.

And at the BE side, you will get url like 'http://127.0.0.1:9300:80', which will make error message like 'port must be a decimal number between 0~65535' which is very weird and hard to diagnose.